dom-renderers/DOMRenderer.js

```diff
@@ -117,6 +117,29 @@ DOMRenderer.prototype.subscribe = function subscribe(type, preventDefault) {
 };
 
 /**
+ * Removes an EventListener of given type from the element on which it was
+ * registered.
+ *
+ * @method
+ *
+ * @param {String} type DOM event type (e.g. click, mouseover).
+ *
+ * @return {undefined} undefined
+ */
+DOMRenderer.prototype.unsubscribe = function unsubscribe(type) {
+    this._assertTargetLoaded();
+    this._target.subscribe[type] = false;
+
+    if (
+        this._target.listeners[type] || this._root.listeners[type]
+    ) {
+        var target = eventMap[type][1] ? this._root : this._target;
+        target.listeners[type] = null;
+        target.element.removeEventListener(type, this._boundTriggerEvent);
```

Comment:
Let's say I'm listening for click events on node A and B. The click event will be delegated, as a result there will only be one listener on the root element. When I'm removing the UI event from node A, you would remove the the event listener from the root element. This means you would no longer receive click events for node B. Therefore the proposed situation is problematic for events that are being delegated.

I would propose to never remove any listeners, but instead simply setting `target.subscribe[type]` to `false`.